my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] **Required:** Great job and very good code. However, you class is missing the `each()` method. Kindly add this method such that when called on an instance of the class, the element(s) is looped through and each 😄 element printed out.

![image](https://user-images.githubusercontent.com/27709832/198531610-667c54a3-4e15-4cdf-bfa8-ddc0e576a224.png)